person.rb

```diff
@@ -0,0 +1,30 @@
+require 'securerandom'
+
+class Person
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = SecureRandom.uuid
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id, :name, :age
```

Comment:
- [ ] **Optional:**
- Conventionally, the `attr` is defined at the top of the class before the initialize method.